pkg/gofr/datasource/dgraph/dgraph.go

```diff
@@ -241,18 +227,18 @@ func (d *Client) Alter(ctx context.Context, op any) error {
 		Duration: duration,
 	}
 
-	span.SetAttributes(
-		attribute.String("dgraph.alter.operation", operation.String()),
-		attribute.Int64("dgraph.alter.duration", duration),
-	)
+	if span != nil {
```

Comment:
I might be wrong, but we are doing it in `line 286`, through `sendOperationStats()` called in `line 241` , here in this function. Is this necessary here ?